packages/appeals-service-api/src/routes/v2/users/service.js

```
@@ -122,6 +123,21 @@ async function linkUserToAppeal(id, appealId, role) {
 	return appealUserRepository.linkUserToAppeal(id, appealId, role);
 }
 
+/**
+ * Unlink user's specified role on an appeal
+ * @param {string} id
+ * @param {string} appealId
+ * @param {AppealToUserRoles} role
+ * @returns {Promise<void>}
+ */
+async function unlinkUserFromAppeal(id, appealId, role) {
+	if (!APPEAL_USER_ROLES_ARRAY.some((appealRole) => appealRole.name === role)) {
```

I would probably throw an error here if the role is anything other than rule6, we don't have a plan for why any other role would come through

---

I've updated the check to error if the role !== rule6Party